scico/linop/radon_svmbir.py

```diff
@@ -56,13 +57,15 @@ def __init__(
             angles: Array of projection angles in radians, should be
                 increasing.
             num_channels: Number of pixels in the sinogram.
+            center_offset: Position of the image center relative to the center-of-rotation, in units of pixels
```

Comment:
Why is this a scalar? As described, I would expect it to be a length-2 vector, because the image is 2D.

Reply:
I think only a scalar needs to be inputted since an offset parallel to the x-ray doesn't change the projector, only offset perpendicular to the ray does.

Reply:
(Response to the comment below, can't seem to reply there)
In the svmbir package, channel-size of the detector and pixel-size can be different but in the scico-svmbir interface, we simplify the interface to make them the same. 
Pixels seem simpler to me since `num_channels` is defined in terms of pixels in the doc-string anyway. But I am open to changing the docstring wording to channels

Reply:
How about I change the parameter description to "Position of the image center relative to the center-of-rotation along the direction perpendicular to the x-rays, in units of pixels"

Reply:
Hm, seems like my complaint is more with svmbir than with your PR. I guess for the PR, so long as it is clear which parameter of svmbir the user is setting (and I think it's already clear), I think it's fine.

Regarding the parameter, your description makes more sense. Maybe it would be even more natural to speak about the offset in the projection domain (maybe you were already trying to do that? I'm reading "image" as object-domain). This would be consistent with the use of "channels" in the svmbir docs. You also get the "perpendicular to the X-rays" part for free.

Reply:
That is a nice idea. Implemented the change in commit ef0a4a4.